### PR TITLE
DISPATCH-1974 Remove flake8 suppressions and fix import issues F401 and F403

### PR DESCRIPTION
**Pull request by jiridanek**

### CODE REVIEW

**Review by jiridanek — COMMENTED** [review]
Lets see what Travis CI thinks.

File: python/qpid_dispatch_internal/display_name/display_name.py [hunk 1]
```diff
@@ -29,14 +29,10 @@
...
-
+import json
 import traceback
-from traceback import format_exc
 
-from ..router.message import Message
-
-from ..dispatch import IoAdapter, LogAdapter, LOG_INFO, LOG_ERROR, LOG_TRACE, LOG_STACK_LIMIT
-import json
+from qpid_dispatch_internal import dispatch
```
[the remark this answers is not in the document]
Comment (ChugR) [reply python/qpid_dispatch_internal/display_name/display_name.py:35]: agreed

File: tests/system_tests_tcp_adaptor.py [hunk 2]
```diff
@@ -61,7 +55,7 @@ class TCP_echo_server(object):
 DISABLE_SELECTOR_TESTS = False
 DISABLE_SELECTOR_REASON = ''
 try:
-    import selectors
+    import selectors  # noqa F401: 'selectors' imported but unused
```
[the remark this answers is not in the document]
Comment (ChugR) [reply tests/system_tests_tcp_adaptor.py:58]: In order to run these tests the echo server and echo client must be run as subprocesses. And those programs use selectors. This test can't work without selectors even though it doesn't use them directly.

**Review by ChugR — APPROVED** [review]
This PR is paying off some technical debt and none of my pending branches will be broken too badly by it.

File: tests/system_tests_tcp_adaptor.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (jiridanek) [reply tests/system_tests_tcp_adaptor.py:58]: I'll update the comment to actually explain what's this about.

Comment (jiridanek) [inline tests/system_tests_tcp_adaptor.py:58]:
```suggestion
    import selectors  # noqa F401: imported but unused (echo server and echo client import this, they run as subprocesses)
```

**Pull request merged**